atest/_resources/keywords/Lab.robot

```diff
@@ -206,7 +206,7 @@ Restart and Run All
     Lab Command    Restart Kernel and Run All Cells
     Accept Default Dialog Option
     Ensure Sidebar Is Closed
-    Wait Until Element Contains    ${JLAB XP LAST CODE PROMPT}    [*]:
+    Run Keyword and Ignore Error    Wait Until Element Contains    ${JLAB XP LAST CODE PROMPT}    [*]:
```

Comment:
this seems to be flaky on windows... might have to be a "wait until keyword succeeds" kind of thing...